server/src/main/java/io/druid/segment/indexing/RealtimeTuningConfig.java

```diff
@@ -40,6 +40,7 @@
 public class RealtimeTuningConfig implements TuningConfig, AppenderatorConfig
 {
   private static final int defaultMaxRowsInMemory = 75000;
+  private static final int defaultMaxOccupationInMemory = 64 << 20;
```

Comment:
I also thought "estimatedOccupation" is basically underestimated value. It's hard to estimate memory usage in java inherently and can different from environments like version, os, etc. "If it cannot be exact, let it have some conservative value by default and make it configurable by users", that I though.